bob-backend/src/pearl/holder.rs

```diff
@@ -79,11 +81,20 @@ impl Holder {
         storage.blobs_count().await
     }
 
+    pub async fn active_index_memory(&self) -> usize {
+        let storage = self.storage.read().await;
+        storage.active_index_memory().await
+    }
+
     pub async fn index_memory(&self) -> usize {
         let storage = self.storage.read().await;
         storage.index_memory().await
     }
 
+    pub async fn has_excess_resources(&self) -> bool {
+        self.index_memory().await > 0
```

Comment:
I think we should not treat active blob indexes as excessive, thus it is better to use `inactive_index_memory` function here